LDK/resources/web/LDK/panel/SingleSubjectFilterType.js

```diff
@@ -189,10 +189,60 @@ Ext4.define('LDK.panel.SingleSubjectFilterType', {
         LABKEY.Query.selectRows(this.aliasTable);
     },
 
+    // This looks for rows that match subjects case insensitively. This is called due to the filter in the case insensitive
+    // case being "contains" instead of "equals" to get all possible casings. This also gets Ids that just contain the
+    // subject id as a substring. Thus we need this to filter out the non-matching rows.
+    getCaseInsensitiveMatches: function(results) {
+        var hasAlias = !!this.aliasTable.aliasColumn;
+        var updatedResults = [];
+
+        Ext4.each(results.rows, function (row) {
+            if (hasAlias) {
```

Comment:
Can we consolidate these two cases? Seems like the only thing that's different is using `this.aliasTable.aliasColumn` vs `this.aliasTable.idColumn` in their first lines

Reply:
Yeah I consolidated them